langchain/src/vectorstores/tests/vectara.int.test.ts

```diff
@@ -130,8 +130,14 @@ describe("VectaraStore", () => {
       const results = await store.similaritySearch(
         "Was Gandalf dead?",
         10, // Number of results needed
-        { lambda: 0.025 }
+        { lambda: 0.025, 
+          contextConfig: {
+          nAfter: 1,
+          nBefore: 1,
+        },
+       }
       );
+      console.log(results[0].pageContent);
```

Comment:
Do you want to leave this or was this just for debug purposes?

Reply:
Yes, I forgot to remove it. This was just for debugging purposes.